rakelib/prod/user_credential.rake

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+desc 'Lock and unlock user credentials'
+namespace :user_credential do
+  task :lock, %i[credential_id type] => :environment do |_, args|
+    log_task_start(namespace: 'UserCredential::Lock', type: args[:type], credential_id: args[:credential_id])
+    set_access_token(credential_id: args[:credential_id], type: args[:type])
+    response = controller_instance.credential_lock
```

Comment:
You can just directly call `lock!` on the `UserVerification` here instead of going through the controller, then you wont need to worry about the access_token stuff

Reply:
I wanted to preserve the normal credential_lock & credential_unlock logging behavior that's present in the controller so that manually locking accounts still trigger "regular" logs (that include the `rake_task` user_identifier) on top of the manual ticket that an engineer will create/update with the logs the raketask puts into the console.